examples/plot_poisson_2.py

```diff
@@ -80,19 +81,27 @@
 #         \\
 #         \exp(\eta) \sum_i \Big(1 - \frac{\lambda_i}{y_i}\Big)x_{ij}, & \text{if}\ z_i \gt \eta
 #     \end{cases}
+#
+#
+#
+
+########################################################
 
 ########################################################
 
 z = np.linspace(0., 10., 100)
 qu = glm_poissonexp.qu(z)
-plt.plot(z, qu)
+plt.plot(z, qu, label='a')
 plt.hold
-plt.plot(z, np.exp(z))
+plt.plot(z, np.exp(z), label='b')
```

Comment:
I wasn't sure on what to properly label them as so I just assigned arbitrary values that can be easily fixed later.